src/test/java/com/wojciechbarwinski/demo/epic_board_games_shop/repositories/OrderCriteriaQueryRepositoryImplTest.java

```diff
@@ -0,0 +1,11 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop.repositories;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OrderCriteriaQueryRepositoryImplTest {
+
+    //TODO wydaje mi się że tą klasę należałoby przetestować, ale jednostkowo raczej nie ma to sensu. Można by te prywatne rozdzielić do osobnej klasy jako pakietowe i tak przetestować tylko je, ale to nie rozwiązuje problemu
```

Comment:
Tak, jednostkowo nie ma sensu - zrób integracyjny. A co do danych to tak jak przegadaliśmy :D jakbyś nie pamiętał to dzwoń :D